toolchain/check/testdata/interop/cpp/no_prelude/function.carbon

```diff
@@ -131,52 +151,61 @@ fn F() {
 // TODO: Test that template functions are unsupported.
 //       This is not tested because template functions are not considered a single result when doing lookup.
 
-// --- unsupported_param_type_function_decl.h
+// ============================================================================
+// param_type_unsupported
+// ============================================================================
+
+// --- param_type_unsupported.h
 
 struct S {};
 
 auto foo(S) -> void;
 
-// --- fail_import_unsupported_param_type_function_decl.carbon
+// --- fail_import_param_type_unsupported.carbon
 
 library "[[@TEST_NAME]]";
 
-import Cpp library "unsupported_param_type_function_decl.h";
+import Cpp library "param_type_unsupported.h";
 
 fn F() {
-  // CHECK:STDERR: fail_import_unsupported_param_type_function_decl.carbon:[[@LINE+7]]:3: error: semantics TODO: `Unsupported: parameter type: struct S` [SemanticsTodo]
+  // CHECK:STDERR: fail_import_param_type_unsupported.carbon:[[@LINE+7]]:3: error: semantics TODO: `Unsupported: parameter type: struct S` [SemanticsTodo]
   // CHECK:STDERR:   Cpp.foo({});
   // CHECK:STDERR:   ^~~~~~~
-  // CHECK:STDERR: fail_import_unsupported_param_type_function_decl.carbon:[[@LINE+4]]:3: note: in `Cpp` name lookup for `foo` [InCppNameLookup]
+  // CHECK:STDERR: fail_import_param_type_unsupported.carbon:[[@LINE+4]]:3: note: in `Cpp` name lookup for `foo` [InCppNameLookup]
   // CHECK:STDERR:   Cpp.foo({});
   // CHECK:STDERR:   ^~~~~~~
   // CHECK:STDERR:
   Cpp.foo({});
 }
 
-// --- non_void_return_function_decl.h
+// ============================================================================
+// return_type_unsupported
```

Comment:
```suggestion
// struct_return_type
```

Reply:
And similarly for the rest of this test.

Reply:
Done.
See my comment in the previous test.